dapps/digital-twin-detail/src/components/container/Container.ts

```diff
@@ -49,7 +49,7 @@ export default class ContainerComponent extends mixins(EvanComponent) {
   get entries() {
     return this.$store.state.container.entryKeys.map((key: string) => ({
       id: key,
-      label: key,
+      label: this.$t(`${this.$route.params.container}.${key}.name`, key),
```

Comment:
shouldn't be the translation be in the template not in the Dapp? Seems wrong to me to force translations on keys by the UI ...

Reply:
The entries are passed into the `evan-onpage-navigation`. Is not supported currently ;)